fix: Remove encoding character issue beside "PowerShell" (#1084)
MicrosoftDocs/windows-powershell-docs: docset/winserver2012r2-ps/adrms/ADRMS.md

# ADRMS Module
## Description

> This topic contains brief descriptions of the Windows PowerShell cmdlets that are for use in deploying in Windows Server 2012. Each cmdlet in the table is linked to additional information about that cmdlet. These cmdlets give you the ability to install, upgrade, or remove a cluster using Windows PowerShell.

Reviewer: ```suggestion
This topic contains brief descriptions of the Windows PowerShell cmdlets used to deploy in Windows Server 2012. Each cmdlet in the table is linked to additional information about that cmdlet. These cmdlets allow you to install, upgrade, or remove a cluster using Windows PowerShell.
```

> This topic contains brief descriptions of the Windows PowerShell cmdlets that are for use in deploying in Windows Server 2012. Each cmdlet in the table is linked to additional information about that cmdlet. These cmdlets give you the ability to install, upgrade, or remove an cluster using Windows PowerShell.

Reviewer 2: ```suggestion
This topic contains brief descriptions of the Windows PowerShell cmdlets that are for use in deploying in Windows Server 2012. Each cmdlet in the table is linked to additional information about that cmdlet. These cmdlets give you the ability to install, upgrade, or remove a cluster using Windows PowerShell.
```
